src/OpenXMLSDK.Engine/Word/ReportEngine/Models/UniformGrid.cs

```diff
@@ -15,6 +15,11 @@ public class UniformGrid : Table
         /// </summary>
         public bool CantSplitRows { get; set; }
 
+        /// <summary>
+        /// Indicate the number of column for spliting the grid 
+        /// </summary>
+        public string ColumnNumber { get; set; }
```

Comment:
string value for a number ?
Please use a nullable int instead

Reply:
I replace ColumnNumber by ColumnNumberKey because it's a key coming from context.